Appraisals

```diff
@@ -1,5 +1,5 @@
 appraise "sass-3-4" do
-  gem "sass", "~> 3.4"
+  gem "sass", "~> 3.4.22"
```

Comment:
Depends, I think. What does administrate need from .22 that .0 doesn't provide? Looks like it's Bourbon? Can the bourbon version be bumped backwards a little bit to accommodate some flexibility in that realm.

If possible, I'd suggest changing the Administrate sass to NOT use any features that are locked into later/younger versions.